README.md

```diff
@@ -3,22 +3,101 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
```

Comment:
I don't think this include is going to work here,   the readme file is not included in docusaurus . 

The readme is mostly for GitHub users

Reply:
Right, then I gonna change the image referencing